### ISSUE TITLE
HPDF_GetFontDef doesn't work with font's real name.

### ISSUE THREAD

**Issue by ghost** [libharu/libharu#65]
When a font is "Bold" or "Italic" or "BoldItalic", the base_font is assigned with 'font name'+'style name', this causes lots of problems.

for example: font Arial Gras, its unique real name is "Arial-BoldMT", which Adobe's software use it as internal font name. but in here the base_font is "Arial-BoldMT,Bold", why add the subfamily name on it? "Arial-BoldMT" is enough.

I wish you can remove the extra part.

**Comment by Aptic-Klas** [comment]
I agree, I don't understand the need for this and in the code base I'm working with, we've removed this.
